CONTRIBUTORS.md

```diff
@@ -3,7 +3,7 @@
 - Arno Baudu
 - Thibaut Chaumont
 - Jennifer Chaignepain
+- Guillaume LAINÉ
```

Comment:
<a href="http://gph.is/1fGyqTb"><img src="https://media.giphy.com/media/ybfSpkMsIvCJW/giphy.gif" border="0" /></a><br />via <a href="http://gph.is/1eS4OkJ">giphy.com</a>